openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1152,14 +1152,16 @@ The path formed following the steps above MUST point to a JSON document complian
 This specification defines the following Credential Issuer Metadata:
 
 * `credential_issuer`: REQUIRED. The Credential Issuer's identifier, as defined in (#credential-issuer-identifier).
-* `authorization_server`: OPTIONAL. Identifier of the OAuth 2.0 Authorization Server (as defined in [@!RFC8414]) the Credential Issuer relies on for authorization. If this element is omitted, the entity providing the Credential Issuer is also acting as the AS, i.e., the Credential Issuer's identifier is used as the OAuth 2.0 Issuer value to obtain the Authorization Server metadata as per [@!RFC8414].
+* `authorization_servers`: OPTIONAL. A JSON array of strings, where each string is an identifier of the OAuth 2.0 Authorization Server (as defined in [@!RFC8414]) the Credential Issuer relies on for authorization. Since Credential Issuer may be using different authorization server depending on the Grant Type, the Wallet should determine which Authorization Server to use using `grant_types_supported` Authorization Server metadata parameter (If omitted, the default value is [`authorization_code`]). If this parameter is omitted, the entity providing the Credential Issuer is also acting as the AS, i.e., the Credential Issuer's identifier is used as the OAuth 2.0 Issuer value to obtain the Authorization Server metadata as per [@!RFC8414].
```

Comment:
This wording feels really specific, whereas https://datatracker.ietf.org/doc/html/draft-ietf-oauth-resource-metadata-00#section-2-2.4 leaves the behaviour quite open and I feel like other people are likely to have use cases for multiple authorization servers other than an AS per grant type.

If we do want to say something then perhaps it should be more along the lines:

"When there are multiple entries in the array the wallet may be able to determine which to use by querying the server metadata, for example depending on the grant type the wallet plans to use examining the `grant_types_supported` values might allow the wallet to eliminate all but one server."

Reply:
good point. 

```suggestion
* `authorization_servers`: OPTIONAL. A JSON array of strings, where each string is an identifier of the OAuth 2.0 Authorization Server (as defined in [@!RFC8414]) the Credential Issuer relies on for authorization. When there are multiple entries in the array, the Wallet may be able to determine which AS to use by querying the metadata, for example, by examining the `grant_types_supported` values might allow the wallet to eliminate all but one server based on the grant type the wallet plans to use. If this parameter is omitted, the entity providing the Credential Issuer is also acting as the AS, i.e., the Credential Issuer's identifier is used as the OAuth 2.0 Issuer value to obtain the Authorization Server metadata as per [@!RFC8414].
```